src/cf.data.pre

```diff
@@ -4979,10 +4979,32 @@ DOC_START
 		[http::]<pt	Peer response time in milliseconds. The timer starts
 				when the last request byte is sent to the next hop
 				and stops when the last response byte is received.
-		[http::]<tt	Total time in milliseconds. The timer
-				starts with the first connect request (or write I/O)
-				sent to the first selected peer. The timer stops
-				with the last I/O with the last peer.
+		[http::]<tt	Total peering time in milliseconds.
+
+				The timer starts when Squid decides to forward the request (to
+				an origin server or cache_peer) and peer selection begins. The
+				timer stops when relevant forwarding activities (including any
+				retries) end.
+
+				Between those two timer events, Squid may perform DNS lookups,
+				query external ACL helpers, adapt responses using pre-cache
+				RESPMOD services, and participate in other concurrent
+				secondary activities. Most secondary activities increase
+				peering time. In some cases, a secondary activity may start
+				before the timer starts or end after the timer stops, leading
+				to misleading results of simple computations like %<tt - %dt.
+
+				If this logformat %code is used before its timer starts, the
+				corresponding measurement has no value (and the %code expands
+				to a single dash ("_") character). If this code is used while
+				its timer is running, the time spent so far is used as the
+				measurement value.
+
+				When Squid re-forwards the request (e.g., due to an HTTP 304
+				response indicating that the cached response is stale), the
+				timer may restart. When a stopped timer restarts, the time
+				passed since the timer was stopped is added to the
+				measurement, as if the timer has not stopped.
```

Comment:
AFAICT, current PR code does not match this documentation because startPeerClock() does nothing if called multiple times, including if it is called after stopPeerClock().